challenges/objects-arrays.js

```diff
@@ -79,9 +106,19 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 {"animal_name":"Australian pelican","population":5,"scientific_name":"Pelecanus conspicillatus","state":"West Virginia"}];
 
 // The zoos need a list of all their animal's names converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
-const lowerCase = [];
 console.log(lowerCase); 
+function getAnimalsTolowercase (arr){
+  const lowerCase = [];
+  arr.map( (val) => lowerCase.push(val.animal_name.toLowerCase())) ;
+  return lowerCase;
+}
+
+console.log(getAnimalsTolowercase (zooAnimals));
 
```

Comment:
Same with all these `.filter()`, `.map()`, `.forEach()`, `.reduce()`. It's good that you used them all properly though.